server/server.go

```diff
@@ -155,16 +155,20 @@ func Serve(efs *embed.FS) {
 		query(),
 	)
 
-	C := rex.Serve(rex.ServerConfig{
+	serverConfig := rex.ServerConfig{
 		Port: uint16(port),
-		TLS: rex.TLSConfig{
+	}
+	if httpsPort > 0 {
+		serverConfig.TLS = rex.TLSConfig{
 			Port: uint16(httpsPort),
 			AutoTLS: rex.AutoTLSConfig{
 				AcceptTOS: !isDev,
```

Comment:
suggestion: `AcceptTOS: httpsPort > 0 && !isDev`

Reply:
can make the code looks better

Reply:
I'm kind of confused about the business logic regarding `AutoTSL`, `AcceptTOS`, and `TSL`. I would expect `AutoTSL` is optional, is it not?